posthog/constants.py

```diff
@@ -89,6 +89,21 @@
 FUNNEL_FROM_STEP = "funnel_from_step"
 FUNNEL_TO_STEP = "funnel_to_step"
 FUNNEL_STEP = "funnel_step"
+FUNNEL_ORDER_TYPE = "funnel_order_type"
+FUNNEL_VIZ_TYPE = "funnel_viz_type"
+
+
+class FunnelOrderType(str, Enum):
+    STRICT = "strict"
+    UNORDERED = "unordered"
+    ORDERED = "ordered"
+
+
+class FunnelVizType(str, Enum):
```

Comment:
Better names very welcome.

I'm still not quite sure how we settled on `ActionsLineGraph` display type for Funnel Trends, but I think making it cleaner like this makes sense^ (thanks Twixes!)